src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  position: relative;
+  width: $catalogWidth;
+  height: $catalogHeight;
+
+  &__card {
+    position: absolute;
+
+    @for $i from 1 through $columnCount {
+      &:nth-child(#{$columnCount}n + #{$i}) {
+        left: getHorizontalShift($cardWidth, $cardShiftHorizontal, $i);
```

Comment:
The function 'getHorizontalShift' is not defined in this file. Make sure to import the file containing this function or define it in this file.

Reply:
The function 'getHorizontalShift' is defined in _functions.scss and I imported this file in main.scss, as in the video tutorial.